app/assets/javascripts/Components/repo_browser.jsx

```diff
@@ -6,6 +6,7 @@ import { SubmissionFileManager } from './submission_file_manager'
 class RepoBrowser extends React.Component {
   constructor(props) {
     super(props);
+    this.submissionFileManagerRef = React.createRef();
     this.state = {
       revision_identifier: props.collected_revision_id,
```

Comment:
Did some more thinking about this one. This line is interesting; I think my original intention was to load the files for the collected revision, while the other other revisions loaded. But this is also overridden when the revisions are loaded to be the most recent revision. You can change this to `undefined`.